### DISCUSSION TITLE
Error loading config file after running soup (module-setup.yml no such file or directory)

### DISCUSSION THREAD

**Discussion by RollingMoses** [Security-Onion-Solutions/securityonion#5491] [Unsupported Versions]
Hello and happy Monday! I updated my lab SO from 2.3.40 to 2.3.70 (standalone) with no errors, so I moved on to my prod instance (distributed) to update from 2.3.61 to 2.3.70. It showed an unhandled exception error at the end of the process. I have seen a couple of others posts related to upgrade errors to 2.3.70, but not the same error. Checking the /root/soup.log shows the following errors, 244 lines of the same:
"Exiting: error loading config file: stat /usr/share/filebeat/module-setup.yml: no such file or directory"
...
"Soup failed with error 1: Unhandled error
Unhandled error occured, please check /root/soup.log for details."

Checking the SOC interface shows that it is running 2.3.70. /etc/soversion also shows 2.3.70. Grid currently shows all but 2 nodes are on 2.3.70 now.
So-status is all green on the manager.

**Comment by dougburks** [comment]
Yeah, I don't think we've seen anything like this in our testing or from others. Were you running a default filebeat configuration or had you modified it?

**Reply by RollingMoses** [reply]
Default filebeat config.

**Reply by RollingMoses** [reply]
Sorry, forgot to add that all nodes are showing 2.3.70 now.

**Reply by dougburks** [reply]
What is the output of `sudo docker images`?

**Reply by RollingMoses** [reply]
REPOSITORY                                                            TAG       IMAGE ID       CREATED        SIZE
ghcr.io/security-onion-solutions/so-thehive-es                        2.3.70    4beff17c0511   5 weeks ago    1.57GB
[hostname_redacted]:5000/security-onion-solutions/so-thehive-es           2.3.70    4beff17c0511   5 weeks ago    1.57GB
ghcr.io/security-onion-solutions/so-influxdb                          2.3.70    37fed31d61a9   5 weeks ago    183MB
[hostname_redacted]:5000/security-onion-solutions/so-influxdb             2.3.70    37fed31d61a9   5 weeks ago    183MB
ghcr.io/security-onion-solutions/so-idstools                          2.3.70    4f33c5d974d1   5 weeks ago    61.6MB
[hostname_redacted]:5000/security-onion-solutions/so-idstools             2.3.70    4f33c5d974d1   5 weeks ago    61.6MB
ghcr.io/security-onion-solutions/so-telegraf                          2.3.70    311dee6c9cab   5 weeks ago    91.5MB
[hostname_redacted]:5000/security-onion-solutions/so-telegraf             2.3.70    311dee6c9cab   5 weeks ago    91.5MB
ghcr.io/security-onion-solutions/so-strelka-manager                   2.3.70    e99b17d426b5   5 weeks ago    21.6MB
[hostname_redacted]:5000/security-onion-solutions/so-strelka-manager      2.3.70    e99b17d426b5   5 weeks ago    21.6MB
ghcr.io/security-onion-solutions/so-redis                             2.3.70    a283e76e62d3   5 weeks ago    32.3MB
[hostname_redacted]:5000/security-onion-solutions/so-redis                2.3.70    a283e76e62d3   5 weeks ago    32.3MB
ghcr.io/security-onion-solutions/so-soc                               2.3.70    1553e77f29b5   5 weeks ago    85.4MB
[hostname_redacted]:5000/security-onion-solutions/so-soc                  2.3.70    1553e77f29b5   5 weeks ago    85.4MB
ghcr.io/security-onion-solutions/so-kratos                            2.3.70    fe64bbfb697d   7 weeks ago    56.1MB
[hostname_redacted]:5000/security-onion-solutions/so-kratos               2.3.70    fe64bbfb697d   7 weeks ago    56.1MB
ghcr.io/security-onion-solutions/so-thehive-cortex                    2.3.70    d3a38490f690   7 weeks ago    1.76GB
[hostname_redacted]:5000/security-onion-solutions/so-thehive-cortex       2.3.70    d3a38490f690   7 weeks ago    1.76GB
ghcr.io/security-onion-solutions/so-zeek                              2.3.70    b35af5727b77   7 weeks ago    573MB
[hostname_redacted]:5000/security-onion-solutions/so-zeek                 2.3.70    b35af5727b77   7 weeks ago    573MB
ghcr.io/security-onion-solutions/so-nginx                             2.3.70    c8ee17546d79   7 weeks ago    87.4MB
[hostname_redacted]:5000/security-onion-solutions/so-nginx                2.3.70    c8ee17546d79   7 weeks ago    87.4MB
ghcr.io/security-onion-solutions/so-grafana                           2.3.70    4dbc4330d4da   7 weeks ago    498MB
[hostname_redacted]:5000/security-onion-solutions/so-grafana              2.3.70    4dbc4330d4da   7 weeks ago    498MB
ghcr.io/security-onion-solutions/so-fleet-launcher                    2.3.70    6693fd38cefe   7 weeks ago    583MB
[hostname_redacted]:5000/security-onion-solutions/so-fleet-launcher       2.3.70    6693fd38cefe   7 weeks ago    583MB
ghcr.io/security-onion-solutions/so-logstash                          2.3.70    afdc45ce9994   7 weeks ago    1.62GB
[hostname_redacted]:5000/security-onion-solutions/so-logstash             2.3.70    afdc45ce9994   7 weeks ago    1.62GB
ghcr.io/security-onion-solutions/so-curator                           2.3.70    e8ea5ec5b542   7 weeks ago    123MB
[hostname_redacted]:5000/security-onion-solutions/so-curator              2.3.70    e8ea5ec5b542   7 weeks ago    123MB
ghcr.io/security-onion-solutions/so-strelka-frontend                  2.3.70    74d2713e295a   7 weeks ago    21.4MB
[hostname_redacted]:5000/security-onion-solutions/so-strelka-frontend     2.3.70    74d2713e295a   7 weeks ago    21.4MB
ghcr.io/security-onion-solutions/so-strelka-backend                   2.3.70    3bfda0b24027   7 weeks ago    1.06GB
[hostname_redacted]:5000/security-onion-solutions/so-strelka-backend      2.3.70    3bfda0b24027   7 weeks ago    1.06GB
ghcr.io/security-onion-solutions/so-wazuh                             2.3.70    a9e256f9c662   7 weeks ago    884MB
[hostname_redacted]:5000/security-onion-solutions/so-wazuh                2.3.70    a9e256f9c662   7 weeks ago    884MB
ghcr.io/security-onion-solutions/so-soctopus                          2.3.70    7147dd65d256   7 weeks ago    509MB
[hostname_redacted]:5000/security-onion-solutions/so-soctopus             2.3.70    7147dd65d256   7 weeks ago    509MB
ghcr.io/security-onion-solutions/so-pcaptools                         2.3.70    d1119d4ec1a8   7 weeks ago    120MB
[hostname_redacted]:5000/security-onion-solutions/so-pcaptools            2.3.70    d1119d4ec1a8   7 weeks ago    120MB
ghcr.io/security-onion-solutions/so-kibana                            2.3.70    125fbdd5c717   7 weeks ago    2.2GB
[hostname_redacted]:5000/security-onion-solutions/so-kibana               2.3.70    125fbdd5c717   7 weeks ago    2.2GB
ghcr.io/security-onion-solutions/so-suricata                          2.3.70    d48491d3d8ad   7 weeks ago    766MB
[hostname_redacted]:5000/security-onion-solutions/so-suricata             2.3.70    d48491d3d8ad   7 weeks ago    766MB
ghcr.io/security-onion-solutions/so-acng                              2.3.70    b3d372fa1357   7 weeks ago    81.9MB
[hostname_redacted]:5000/security-onion-solutions/so-acng                 2.3.70    b3d372fa1357   7 weeks ago    81.9MB
ghcr.io/security-onion-solutions/so-mysql                             2.3.70    f0b736914d3c   7 weeks ago    250MB
[hostname_redacted]:5000/security-onion-solutions/so-mysql                2.3.70    f0b736914d3c   7 weeks ago    250MB
ghcr.io/security-onion-solutions/so-strelka-filestream                2.3.70    d1a064f34418   7 weeks ago    20.6MB
[hostname_redacted]:5000/security-onion-solutions/so-strelka-filestream   2.3.70    d1a064f34418   7 weeks ago    20.6MB
ghcr.io/security-onion-solutions/so-steno                             2.3.70    8ff7eed5eb03   7 weeks ago    849MB
[hostname_redacted]:5000/security-onion-solutions/so-steno                2.3.70    8ff7eed5eb03   7 weeks ago    849MB
ghcr.io/security-onion-solutions/so-thehive                           2.3.70    cc9798752eff   7 weeks ago    631MB
[hostname_redacted]:5000/security-onion-solutions/so-thehive              2.3.70    cc9798752eff   7 weeks ago    631MB
ghcr.io/security-onion-solutions/so-playbook                          2.3.70    d1441ed8bd43   7 weeks ago    604MB
[hostname_redacted]:5000/security-onion-solutions/so-playbook             2.3.70    d1441ed8bd43   7 weeks ago    604MB
ghcr.io/security-onion-solutions/so-elasticsearch                     2.3.70    7b6d0a4b011e   7 weeks ago    1.57GB
[hostname_redacted]:5000/security-onion-solutions/so-elasticsearch        2.3.70    7b6d0a4b011e   7 weeks ago    1.57GB
ghcr.io/security-onion-solutions/so-elastalert                        2.3.70    eb5fcb0ba76a   7 weeks ago    434MB
[hostname_redacted]:5000/security-onion-solutions/so-elastalert           2.3.70    eb5fcb0ba76a   7 weeks ago    434MB
ghcr.io/security-onion-solutions/so-fleet                             2.3.70    80cd1618ac13   7 weeks ago    83.1MB
[hostname_redacted]:5000/security-onion-solutions/so-fleet                2.3.70    80cd1618ac13   7 weeks ago    83.1MB
ghcr.io/security-onion-solutions/so-filebeat                          2.3.70    231155ee4a4b   7 weeks ago    628MB
[hostname_redacted]:5000/security-onion-solutions/so-filebeat             2.3.70    231155ee4a4b   7 weeks ago    628MB
ghcr.io/security-onion-solutions/so-soc                               2.3.61    ba6a8e2083c8   2 months ago   85.3MB
[hostname_redacted]:5000/security-onion-solutions/so-soc                  2.3.61    ba6a8e2083c8   2 months ago   85.3MB
ghcr.io/security-onion-solutions/so-kibana                            2.3.61    25782affded5   2 months ago   2.2GB
[hostname_redacted]:5000/security-onion-solutions/so-kibana               2.3.61    25782affded5   2 months ago   2.2GB
ghcr.io/security-onion-solutions/so-thehive-es                        2.3.61    e2ebf076ee0c   2 months ago   1.57GB
[hostname_redacted]:5000/security-onion-solutions/so-thehive-es           2.3.61    e2ebf076ee0c   2 months ago   1.57GB
ghcr.io/security-onion-solutions/so-logstash                          2.3.61    4599fec6fecc   2 months ago   1.62GB
[hostname_redacted]:5000/security-onion-solutions/so-logstash             2.3.61    4599fec6fecc   2 months ago   1.62GB
ghcr.io/security-onion-solutions/so-elasticsearch                     2.3.61    4369751819e8   2 months ago   1.57GB
[hostname_redacted]:5000/security-onion-solutions/so-elasticsearch        2.3.61    4369751819e8   2 months ago   1.57GB
ghcr.io/security-onion-solutions/so-filebeat                          2.3.61    8d82b549386c   2 months ago   628MB
[hostname_redacted]:5000/security-onion-solutions/so-filebeat             2.3.61    8d82b549386c   2 months ago   628MB
ghcr.io/security-onion-solutions/so-zeek                              2.3.61    889084fd208b   2 months ago   573MB
[hostname_redacted]:5000/security-onion-solutions/so-zeek                 2.3.61    889084fd208b   2 months ago   573MB
ghcr.io/security-onion-solutions/so-thehive-cortex                    2.3.61    3ba0702a9f1b   2 months ago   1.24GB
[hostname_redacted]:5000/security-onion-solutions/so-thehive-cortex       2.3.61    3ba0702a9f1b   2 months ago   1.24GB
ghcr.io/security-onion-solutions/so-strelka-backend                   2.3.61    80d7d30f3000   2 months ago   1.06GB
[hostname_redacted]:5000/security-onion-solutions/so-strelka-backend      2.3.61    80d7d30f3000   2 months ago   1.06GB
ghcr.io/security-onion-solutions/so-wazuh                             2.3.61    10f29b472b6c   2 months ago   884MB
[hostname_redacted]:5000/security-onion-solutions/so-wazuh                2.3.61    10f29b472b6c   2 months ago   884MB
ghcr.io/security-onion-solutions/so-steno                             2.3.61    a3bf7460b9f8   2 months ago   849MB
[hostname_redacted]:5000/security-onion-solutions/so-steno                2.3.61    a3bf7460b9f8   2 months ago   849MB
ghcr.io/security-onion-solutions/so-strelka-filestream                2.3.61    b1684189f393   2 months ago   20.6MB
[hostname_redacted]:5000/security-onion-solutions/so-strelka-filestream   2.3.61    b1684189f393   2 months ago   20.6MB
ghcr.io/security-onion-solutions/so-redis                             2.3.61    60ecbc0f3fd8   2 months ago   32.3MB
[hostname_redacted]:5000/security-onion-solutions/so-redis                2.3.61    60ecbc0f3fd8   2 months ago   32.3MB
ghcr.io/security-onion-solutions/so-kratos                            2.3.61    559557ace3b6   2 months ago   56.1MB
[hostname_redacted]:5000/security-onion-solutions/so-kratos               2.3.61    559557ace3b6   2 months ago   56.1MB
[hostname_redacted]:5000/security-onion-solutions/so-elastalert           2.3.61    1b2f6373c556   2 months ago   434MB
ghcr.io/security-onion-solutions/so-elastalert                        2.3.61    1b2f6373c556   2 months ago   434MB
ghcr.io/security-onion-solutions/so-mysql                             2.3.61    86990ddb2f87   2 months ago   250MB
[hostname_redacted]:5000/security-onion-solutions/so-mysql                2.3.61    86990ddb2f87   2 months ago   250MB
ghcr.io/security-onion-solutions/so-strelka-manager                   2.3.61    703a79d290db   2 months ago   21.2MB
[hostname_redacted]:5000/security-onion-solutions/so-strelka-manager      2.3.61    703a79d290db   2 months ago   21.2MB
[hostname_redacted]:5000/security-onion-solutions/so-fleet                2.3.61    922222624adc   2 months ago   83.1MB
ghcr.io/security-onion-solutions/so-fleet                             2.3.61    922222624adc   2 months ago   83.1MB
ghcr.io/security-onion-solutions/so-thehive                           2.3.61    e983e0f7126c   2 months ago   631MB
[hostname_redacted]:5000/security-onion-solutions/so-thehive              2.3.61    e983e0f7126c   2 months ago   631MB
ghcr.io/security-onion-solutions/so-nginx                             2.3.61    ebf7591b8bea   2 months ago   87.4MB
[hostname_redacted]:5000/security-onion-solutions/so-nginx                2.3.61    ebf7591b8bea   2 months ago   87.4MB
ghcr.io/security-onion-solutions/so-suricata                          2.3.61    399c27ac246e   2 months ago   764MB
[hostname_redacted]:5000/security-onion-solutions/so-suricata             2.3.61    399c27ac246e   2 months ago   764MB
ghcr.io/security-onion-solutions/so-soctopus                          2.3.61    314e61fddfd1   2 months ago   603MB
[hostname_redacted]:5000/security-onion-solutions/so-soctopus             2.3.61    314e61fddfd1   2 months ago   603MB
ghcr.io/security-onion-solutions/so-influxdb                          2.3.61    dbdfac6d4ccb   2 months ago   183MB
[hostname_redacted]:5000/security-onion-solutions/so-influxdb             2.3.61    dbdfac6d4ccb   2 months ago   183MB
ghcr.io/security-onion-solutions/so-playbook                          2.3.61    88c3291e3a07   2 months ago   630MB
[hostname_redacted]:5000/security-onion-solutions/so-playbook             2.3.61    88c3291e3a07   2 months ago   630MB
ghcr.io/security-onion-solutions/so-acng                              2.3.61    d8410fec53a1   2 months ago   81.9MB
[hostname_redacted]:5000/security-onion-solutions/so-acng                 2.3.61    d8410fec53a1   2 months ago   81.9MB
ghcr.io/security-onion-solutions/so-curator                           2.3.61    373826a118fd   2 months ago   123MB
[hostname_redacted]:5000/security-onion-solutions/so-curator              2.3.61    373826a118fd   2 months ago   123MB
[hostname_redacted]:5000/security-onion-solutions/so-fleet-launcher       2.3.61    f010764367b5   2 months ago   583MB
ghcr.io/security-onion-solutions/so-fleet-launcher                    2.3.61    f010764367b5   2 months ago   583MB
[hostname_redacted]:5000/security-onion-solutions/so-strelka-frontend     2.3.61    25da100ce3f6   2 months ago   21.4MB
ghcr.io/security-onion-solutions/so-strelka-frontend                  2.3.61    25da100ce3f6   2 months ago   21.4MB
ghcr.io/security-onion-solutions/so-pcaptools                         2.3.61    7a9b945e5a78   2 months ago   120MB
[hostname_redacted]:5000/security-onion-solutions/so-pcaptools            2.3.61    7a9b945e5a78   2 months ago   120MB
ghcr.io/security-onion-solutions/so-grafana                           2.3.61    f5a5e4a4e930   2 months ago   498MB
[hostname_redacted]:5000/security-onion-solutions/so-grafana              2.3.61    f5a5e4a4e930   2 months ago   498MB
ghcr.io/security-onion-solutions/so-idstools                          2.3.61    10dfa9977e93   2 months ago   56.8MB
[hostname_redacted]:5000/security-onion-solutions/so-idstools             2.3.61    10dfa9977e93   2 months ago   56.8MB
ghcr.io/security-onion-solutions/so-telegraf                          2.3.61    556c1409a918   2 months ago   91.5MB
[hostname_redacted]:5000/security-onion-solutions/so-telegraf             2.3.61    556c1409a918   2 months ago   91.5MB
ghcr.io/security-onion-solutions/registry                             latest    1fd8e1b0bb7e   5 months ago   26.2MB

**Reply by RollingMoses** [reply]
[so_manager_sudo_docker_images_output_20210927.txt](https://github.com/Security-Onion-Solutions/securityonion/files/7236542/so_manager_sudo_docker_images_output_20210927.txt)

**Reply by dougburks** [reply]
I'm not seeing any glaring issues in that output. You might try comparing your two deployments side-by-side and looking for any differences, specifically in the filebeat config.
